src/OpenTelemetry.DependencyInjection/README.md

```diff
@@ -0,0 +1,25 @@
+# OpenTelemetry .NET DependencyInjection
```

Comment:
Should we call this project `OpenTelemetry.Extensions.DependencyInjection`?

Reply:
Related to #3469 (specifically the section titled "Integrations for specific application frameworks or libraries")

I agree, I like `OpenTelemetry.Extensions.DependencyInjection`. The package is mostly extensions involving `IServiceCollection` and I'm pretty sure we don't see a future where it would deviate from this. There are a few extensions from `IServiceProvider` as well, but I think this is ok.

Reply:
Updated with rename to: `OpenTelemetry.Extensions.DependencyInjection`